README.md

```diff
@@ -1,17 +1,17 @@
 To use this package simply: 
 
 ```
- go get https://github.com/content-services/utilities
+ go get github.com/content-services/yummy
 ```
 
 ```
 import (
-	"github.com/content-services/utilities/Yum"
+	"github.com/content-services/yummy/yum"
 )
 ```
 and utilize the module methods:
 
 ```
-	Rpm.Extract("https://download-i2.fedoraproject.org/pub/epel/7/x86_64/")
+	yum.ExtractPackageData(http.Client{}, "https://download-i2.fedoraproject.org/pub/epel/7/x86_64/")
```

Comment:
note that this assume that this pr is going in: https://github.com/content-services/yummy/pull/5